### DISCUSSION TITLE
Error while update to 1.49

### DISCUSSION THREAD

**Discussion by Mkay505** [zwave-js/zwave-js#3786] [Q&A]
Trying to install 0.1.49

Home Assistant 2021.11.5 core
Supervisor 2021.10.8
Host unRAID 6.9.2 (Dell ECM Server)
Zwave JS 0.1.48

Error Message:

Can't install homeassistant/amd64-addon-zwave_js:0.1.49: 404 Client Error for http+docker://localhost/v1.41/images/create?tag=0.1.49&fromImage=homeassistant%2Famd64-addon-zwave_js: Not Found ("manifest for homeassistant/amd64-addon-zwave_js:0.1.49 not found: manifest unknown: manifest unknown"

Any ideas? Thank you in advance

**Comment by AlCalzone** [comment]
This belongs in the Home Assistant repository. I don't have the link handy but you should be able find it.

**Comment by AlCalzone** [comment]
You're looking for the homeassistant/core repo. This is just the zwave library that's used by HA.
